SearchFunction

teonikolov2002 opened the pull request:
Added SearchHandler class, with which the entries in the log instance can be searched by type, date and name of person, who added them. Name search can be exact / partial and just the first characters of the name.

teonikolov2002 committed:
SearchFunction

VillySlavova reviewed (changes requested):
add menuhandler.cs and json realization

teonikolov2002 committed:
Saving Entries from Log in file and ability to load entries from file - added to BuildingLog class

teonikolov2002 commented:
In  last commit I added a LogStorage class for saving the entries from the log instance to a file using JSON Serialization. I have added the option for the entries to be loaded from the file as well. This required adding a new method in BuildingLog.cs
